source/blog/2016-10-28-next-london-2016-google-cloud-platform-s-user-conference.md

```diff
@@ -2,9 +2,9 @@
 date: '2016-10-28 10:39 +0100'
 published: true
 title: NEXT London 2016 - Google Cloud Platform's User Conference
-authors: 
+author: Charlie Egan
+coauthors:
```

Comment:
Why the split from author into author + coauthors ?  It implies a hierarchy of authorship that we might not intend (how to decide who is the author and who is the coauthor, especially meaningless as we don't distinguish in the page rendering).  I'd be in favour of saying we don't have author anymore, just authors (and do a grep to fix all the old frontmatter).